lib/resources.rb

```diff
@@ -877,259 +877,390 @@ def get_resource(key)
+                                                             "name"=> "status",
+                                                             "type"=> "token",
+                                                             "documentation"=> "The status of the prescription"
+                                                         }, {
+                                                             "name"=> "scheduledtiming-bounds-end",
+                                                             "type"=> "date",
+                                                             "documentation"=> "The period end of the prescription timing schedule. This parameter value should be prefixed by \u003c="
+                                                         }, {
+                                                             "name"=> "_count",
+                                                             "type"=> "number",
+                                                             "documentation"=> "The maximum number of results to return"
+                                                         }]
+                                    }, {
+                                        "type"=> "Observation",
+                                        "interaction"=> [{
+                                                             "code"=> "search-type"
+                                                         }],
+                                        "searchParam"=> [{
+                                                             "name"=> "patient",
+                                                             "type"=> "reference",
+                                                             "documentation"=> "The subject that the observation is about (if patient)"
+                                                         }, {
+                                                             "name"=> "subject",
+                                                             "type"=> "reference",
+                                                             "documentation"=> "The subject that the observation is about"
+                                                         }, {
+                                                             "name"=> "code",
+                                                             "type"=> "token",
+                                                             "documentation"=> "The type of observation"
+                                                         }, {
+                                                             "name"=> "date",
+                                                             "type"=> "date",
+                                                             "documentation"=> "The date range into which the observation falls. This parameter value should be present twice, prefixed once by '\u003e' representing the earliest date and once by '\u003c' representing the latest date."
```

Comment:
Dennis mentioned this for content/may2015.md but noting that \u003e and \u003c need to be changed to > and < here too